core/src/main/java/org/apache/spark/api/java/StorageLevels.java

```diff
@@ -40,19 +40,6 @@
    * Create a new StorageLevel object.
    * @param useDisk saved to disk, if true
    * @param useMemory saved to memory, if true
-   * @param deserialized saved as deserialized objects, if true
-   * @param replication replication factor
-   */
-  @Deprecated
```

Comment:
Although not stated along with the `@Deprecated` annotation, this method has been deprecated since April, 2014; see https://github.com/apache/spark/commit/b50ddfde0342990979979e58348f54c10b500c90